test/components/tweets/display_test.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+module Tweets
+  class DisplayTest < ViewComponent::TestCase
+    include Motion::TestHelpers
+
+    let(:klass) { Tweets::Display }
+    let(:content) { "My First Tweet" }
+    let(:tweet) { Tweet.new(content: content) }
+    subject { klass.new(tweet: tweet) }
+
+    describe "rendering" do
+      before do
+        render_inline(subject)
+      end
+
+      it "shows username, svg, and full name in header" do
+        assert_selector ".tweet-header" do
```

Comment:
I was like "what would you need to test about the display?" and it makes sense now, I'm gonna have to remember this :+1: